compat/src/index.js

```diff
@@ -179,7 +178,6 @@ function normalizeVNode(vnode) {
 function cloneElement(element) {
 	if (!isValidElement(element)) return element;
 	let vnode = normalizeVNode(preactCloneElement.apply(null, arguments));
-	vnode.$$typeof = REACT_ELEMENT_TYPE;
```

Comment:
This is already set in `options.vnode` hook which `preactCloneElement` calls so no need to reset here I think